.github/workflows/pre-release-continuous-test.yaml

```diff
@@ -0,0 +1,19 @@
+name: Run continuous pre release testing
+on:
+  schedule:
+    # Run hourly
+    - cron: "0 * * * *"
+
+permissions:
+  contents: read
+  id-token: write #required for GCP Workload Identity federation which we use to login into Google Artifact Registry
```

Comment:
do we need this?

Reply:
contents give access to pr, i guess we dont need the other it was present in another file